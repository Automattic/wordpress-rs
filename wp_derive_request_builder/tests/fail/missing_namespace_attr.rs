#[derive(wp_derive_request_builder::WpDerivedRequest, serde::Serialize)]
#[SparseField(crate::SparseUserField)]
enum UsersRequest {
    #[contextual_get(url = "/users", params = &UserListParams, output = Vec<SparseUser>)]
    List,
}

fn main() {}
