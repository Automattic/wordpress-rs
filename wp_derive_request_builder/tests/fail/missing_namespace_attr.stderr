error: Missing #[Namespace("_path_")] attribute
 --> tests/fail/missing_namespace_attr.rs:3:1
  |
3 | enum UsersRequest {
  | ^^^^
